chunky/src/java/se/llbit/chunky/ui/render/tabs/GeneralTab.java

```diff
@@ -140,6 +133,11 @@ public GeneralTab() throws IOException {
       loadOtherEntities.setSelected(preferences.shouldLoadClass(null));
     }
     biomeColors.setSelected(scene.biomeColorsEnabled());
+    if(!scene.biomeColorsEnabled()) {
+      biomeBlending.setDisable(true);
+    }
```

Comment:
Does this get updated if biome colors are enabled later?

```suggestion
    biomeBlending.setDisable(!scene.biomeColorsEnabled());
```

Reply:
Yes, this is just the initial value, see https://github.com/chunky-dev/chunky/pull/1225/files/01941b8ce6f2a018dd32920d7cd5b548c27a688d#diff-508504c789f61f4751ea96ead9517f30b1a68bbb68a7fecade06614ff5d37575R275

Reply:
Though I agree with your logic :+1: